src/source-af-packet.c

```diff
@@ -77,19 +76,6 @@
 #include <bpf/bpf.h>
 #endif
 
-struct bpf_program {
-    unsigned int bf_len;
-    struct bpf_insn *bf_insns;
-};
```

Comment:
Seems that we need this one? https://github.com/OISF/suricata/actions/runs/8652209800/job/23724605678#step:12:54

Reply:
We should not, but this is include hell :
`bpf_program` may be defined by both kernel bpf.h and by libpcap pcap.h (and other like bpf_insn)
So, we should only be using either pcap or bpf in each C file, but this is not the case here

Reply:
bpf.h is needed for `bpf_map_update_elem(mapd, key, value, BPF_NOEXIST)`

pcap.h is needed for `PCAP_ERRBUF_SIZE` and `SCBPFCompile`